### PR TITLE
fix: show dashboard tab with ES consumer

### PR DESCRIPTION
**Pull request by obdulia-losantos**
Hide the Dashboard tab in the Survey page if the ES consumer is not connected to Gather, when the `ES_CONSUMER_URL` is empty.

### CODE REVIEW

File: app/gather/settings.py [hunk 1]
```diff
@@ -69,6 +69,9 @@
     if key.startswith(AETHER_PREFIX)
 ]
 
+ES_CONSUMER_URL = os.environ.get('ES_CONSUMER_URL')
```
Comment (lordmallam) [inline app/gather/settings.py:73]: Can you include docs for `ES_CONSUMER_URL` environment variable? 📚

Comment (obdulia-losantos) [reply app/gather/settings.py:73]: :writing_hand:

Comment (obdulia-losantos) [reply app/gather/settings.py:73]: @lordmallam Done!

Comment (lordmallam) [reply app/gather/settings.py:73]: Actually meant in `Read Me` with other environment variables. But I guess this is fine for now..

**Review by lordmallam — DISMISSED** [review]

File: app/gather/settings.py (hunk 1 above)
Comment (obdulia-losantos) [reply app/gather/settings.py:73]: :joy: 

You are right, but we can add it in the README once we finish the feature :thinking:

Comment (obdulia-losantos) [reply app/gather/settings.py:73]: @lordmallam in the right place! :nerd_face:

**Review by lordmallam — APPROVED** [review]

**Pull request merged**